assets/icons/bufferingIcon.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="17px" height="17px" viewBox="0 0 17 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 57.1 (83088) - https://sketch.com -->
+    <title>buffering</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
Come nell'altra PR, dovremmo evitare di aggiungere metadati ai file SVG: https://github.com/svg/svgo

Reply:
Ma se nella pipeline mettiamo quel pacchetto che suggerivi che ottimizza le SVG le leva da solo? Perché sketch me le esporta in automatico con i metadati e per workflow è un po' una pezza se devo levare a mano i metadati ogni volta.